.github/scripts/install-torch-tensorrt.sh

```diff
@@ -2,8 +2,8 @@
 set -eou pipefail
 # Source conda so it's available to the script environment
 source ${BUILD_ENV_FILE}
-${CONDA_RUN} ${PIP_INSTALL_TORCH} torchvision pyyaml
+${CONDA_RUN} ${PIP_INSTALL_TORCH} torch==2.1.2 torchvision==0.16.2 pyyaml
```

Comment:
does this do anything? since theres already ${PIP_INSTALL_TORCH}?

Reply:
This is necessary, since in `build-test.yml`, when building Torch-TRT, we have:
```
PIP_INSTALL_TORCH='pip install torch==2.1.2 --extra-index-url https://download.pytorch.org/whl/test/cu118'
```
In `linux-test.yml`, when testing Torch-TRT using the artifact from build, we have:
```
PIP_INSTALL_TORCH='pip install torch --extra-index-url https://download.pytorch.org/whl/test/cu118'
```
The second one installs the `2.2.0.dev` test build, which is the most up-to-date test build, but is incompatible with the version on which it was built. Specifying the version explicitly resolves this problem.